lib/distributions/Circle/FourierDistribution.m

```diff
@@ -705,6 +705,18 @@
                         warning('Conversion:NoFormula', 'No explicit formula available, using FFT to get transformation');
                         f = FourierDistribution.fromFunction(@distribution.pdf, noOfCoefficients, desiredTransformation);
                     end
+                case 'FIGDistribution'
+                    % This also works if not right number as it will be

```

Comment:
Can you add what `right number`  means here? Even number?

Reply:
Improved the comment to make clear that I meant that the number of grid points does not need to be equal to the number of Fourier coefficients.